spacy_transformers/language.py

```diff
@@ -84,7 +84,8 @@ def update(self, docs, golds, drop=0.0, sgd=None, losses=None, component_cfg={})
         for doc, gold in zip(docs, golds):
             if isinstance(doc, str):
                 doc = self.make_doc(doc)
-                doc = sentencizer(doc)
+            doc = sentencizer(doc)
```

Comment:
To make this a smidge more consistent-looking (although in effect there's no difference for the current sentencizer), I'd suggest:

```python
if not doc.is_sentenced:
    doc = sentencizer(doc)
```

Reply:
Hi @adrianeboyd . I actually noticed that this is fixed in  WIP NER branch https://github.com/explosion/spacy-transformers/pull/95/commits/8cf378b7017a65a806e93e346a69b11f271173cf. Is this pull request still necessary?

Reply:
I doubt that branch will be merged in the end (Matt's now working on a separate rewrite for spacy v3), so it makes sense to fix this here.